Products/ZenModel/data/devices.xml

```diff
@@ -11271,7 +11271,7 @@ True
 False
 </property>
 <property type="string" id="maxval" mode="w" >
-long(here.speed) / 8 * .75
+here.speed / 8 * .75
```

Comment:
I surprised to see these formula changes.  I suppose there's a migration script that wasn't accompanied by updates to the XML and SQL dump files.